integration_tests/pages/manage/booking/show.ts

```diff
@@ -16,8 +16,8 @@ export default class BookingShowPage extends Page {
 
   shouldShowBookingDetails(booking: Booking): void {
     cy.get('dl[data-cy-expected-dates]').within(() => {
```

Comment:
should we update this attribute too?

Reply:
I think it's fine, as far as the user is concerned they're expected dates. We just need to make the internal naming consistent

Reply:
Actually, as this data attribute is an internal thing, it probably does make sense to be consistent here too! 😆